core/blockchain/impl/key_value_block_storage.hpp

```diff
@@ -17,55 +17,24 @@ namespace kagome::blockchain {
 
   class KeyValueBlockStorage : public BlockStorage {
    public:
-    using BlockHandler = std::function<void(const primitives::Block &)>;
-
-    enum class Error {
-      BLOCK_EXISTS = 1,
-      HEADER_DOES_NOT_EXIST,
-      BODY_DOES_NOT_EXIST,
-      BLOCK_DATA_DOES_NOT_EXIST,
-      JUSTIFICATION_DOES_NOT_EXIST,
-      GENESIS_BLOCK_ALREADY_EXISTS,
-      GENESIS_BLOCK_NOT_FOUND,
-      FINALIZED_BLOCK_NOT_FOUND,
-    };
-
     ~KeyValueBlockStorage() override = default;
 
-    static outcome::result<std::shared_ptr<KeyValueBlockStorage>> create(
-        storage::trie::RootHash state_root,
-        const std::shared_ptr<storage::BufferStorage> &storage,
-        const std::shared_ptr<crypto::Hasher> &hasher,
-        const BlockHandler &on_finalized_block_found);
-
     /**
-     * Initialise block storage with existing data
+     * Creates block storage. Iff block storage is empty, then initializes with
+     * a genesis block which is created inside from merkle trie root
```

Comment:
>created inside


inside of what?
poor wording

If you tried to say "inside that method" then the possible substitutes are:
* under the hood
* automatically

Reply:
Well notice. Thanks